static/js/recent_topics_ui.js

```diff
@@ -669,8 +669,14 @@ export function hide() {
     // before it completely re-rerenders.
     message_view_header.render_title_area();
 
-    // Fire our custom event
-    $("#message_feed_container").trigger("message_feed_shown");
+    // We use a requestAnimationFrame here to prevent this
+    // call from causing a forced reflow ie let the browser
+    // optimize by performing this action just before the
+    // next paint.
+    window.requestAnimationFrame(() => {
+        // Fire our custom event
+        $("#message_feed_container").trigger("message_feed_shown");
+    });
```

Comment:
Can the `requestAnimationFrame` be inside the handler for `message_feed_shown`?

Reply:
Made the change. Definitely better now.